docs/USING_PUBLIC_CLOUDS.rst

```diff
@@ -113,7 +113,7 @@ There are a few configurable options here:
 ^^^^^^^^^^^^^^^^^^
 
 .. TIP::
-   If you explicitly set your :code:`accessKey` and :code:`secretKey` in the 
+   If you explicitly set your :code:`accessKey` and :code:`secretKey` in the
```

Comment:
Informative, trivial: Not important, but usually we try to avoid editing whitespaces when not relevant for our code, as it creates noise in the diff + review process :)

Reply:
noted :)